src/vochain/vochain.proto

```diff
@@ -191,7 +211,7 @@ enum ProcessStatus {
 }
 
 enum SourceNetworkId {
-	UNKNOWN = 0;
+	SOURCE_NETWORK_ID_UNKNOWN = 0;
```

Comment:
Updated names of `UNKNOWN` enums to mach the name of the enum to avoid possible conflicts with other enums of similar names.

Reply:
That's probably breaking backwards compatibility, @mvdan can you confirm?

If so, I'd add an issue to change that in the moment we break compatibility (should be soon) and make all breaking changes at once.

Reply:
Note: after talking with @p4u , we will keep this PR just for adding the CensusType related stuff & arbo, without changing the already existing enums, to keep backwards compatibility.
And in a near future we will do another PR with a bit of renaming and refactor, breaking the backwards compatibility.

Reply:
Created an issue to track this future changes https://github.com/vocdoni/dvote-protobuf/issues/19 , that are in a separated branch (https://github.com/vocdoni/dvote-protobuf/tree/feature/vochain-enums-renames).

Reply:
The names of enum values in protobuf do not affect backwards compatibility of the *encoding*, so you can change them safely.

They just change the API of the generated code, so they require fairly simple changes in our code like go-dvote and dvote-js.

For example, for https://github.com/vocdoni/dvote-protobuf/pull/14 we did these changes: https://github.com/vocdoni/vocdoni-node/commit/bc055be1b98298ba16c345a6a3aca4c3e35222dc